src/configure

```diff
@@ -212,13 +212,17 @@ function linux_configure_mkl_libraries {
       local libfile=$mkllibdir/lib$file.so
       check_exists $libfile
     done 
-    echo  "-L$mkllibdir -Wl,-rpath=$mkllibdir -lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core -liomp5 -lpthread -lm"
+    #echo  "-L$mkllibdir -Wl,-rpath=$mkllibdir -lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core -liomp5 -lpthread -lm"
+    linkline="-L$mkllibdir -Wl,-rpath=$mkllibdir  -lmkl_intel_lp64 -lmkl_core -lmkl_tbb_thread -ltbb -lstdc++ -lpthr$
```

Comment:
BTW, looking at the link line advisor, the 11.3 still does support intel
OMP (iomp)...
y.

On Thu, Aug 27, 2015 at 4:45 PM, Angel Castro Martinez <
notifications@github.com> wrote:

> In src/configure
> <https://github.com/kaldi-asr/kaldi/pull/99#discussion_r38143895>:
>
> > @@ -212,13 +212,17 @@ function linux_configure_mkl_libraries {
> >        local libfile=$mkllibdir/lib$file.so
> >        check_exists $libfile
> >      done
> > -    echo  "-L$mkllibdir -Wl,-rpath=$mkllibdir -lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core -liomp5 -lpthread -lm"
> > +    #echo  "-L$mkllibdir -Wl,-rpath=$mkllibdir -lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core -liomp5 -lpthread -lm"
> > +    linkline="-L$mkllibdir -Wl,-rpath=$mkllibdir  -lmkl_intel_lp64 -lmkl_core -lmkl_tbb_thread -ltbb -lstdc++ -lpthr$
>
> Thanks fro noticing
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/99/files#r38143895>.
>